CONTRIBUTING.md

```diff
@@ -138,7 +138,7 @@ which combinations are tested regularly.
    * `CXX=clang++ CC=clang` to use Clang.
    * `CXX=g++ CC=gcc` to use GCC.
  * `DISTRO`: the Linux distribution, use `ubuntu`, `fedora`, or `centos`.
- * `DISTRO_VERSION`: the version of the distribution, e.g. `17.04`.
+ * `DISTRO_VERSION`: the version of the distribution, e.g. `16.04`.
```

Comment:
shouldn't it be 17.10

Reply:
Either really, I chose 16.04 because it is going to work until 2021.  17.10 will become EOL on 2018-07.